src/main/webapp/components/api/api.js

```diff
@@ -9,4 +9,5 @@ angular.module('managementConsole.api', [
   'ispn.directives.cache.cacheconfiguration',
   'ispn.directives.cache.configurationsection',
   'ispn.directives.cache.jdbcstore',
+  'LocalStorageModule'
```

Comment:
@tristantarrant thanks, that sounds good! I investigated it a little bit and sessionStorage is localized per tab/window while localStorage is persistent, which makes sessionStorage a better choice for storing credentials. However, user will have to log in each time he opens a new tab/window - I guess that is ok in this case.
Also, we should keep in mind here that this is still not secure - for real security we should use ssl connection, tokens/session/cookies instead of credentials, use proper cors restrictions - I am not expert on this so there is certainly more things to watch out for, and in the future it would be good to study this thoroughly.

I changed storage type from localStorage to sessionStorage. I also confirmed that localStorageService has 'sessionStorage' set as its type, and that it is supported by my browser, however I could was not able to get the behaviour that I expected - when I opened infinispan console in another window, I was logged in. I am not sure, but I think it may be due to the BrowserSync syncing these tabs.